x/rollapp/client/cli/query_state_info.go

```diff
@@ -51,64 +53,50 @@ func CmdListStateInfo() *cobra.Command {
 
 func CmdShowStateInfo() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   fmt.Sprintf("show-rollapp-state-info [%s] [%s|%s]", FlagRollappId, FlagStateIndex, FlagRollappHeight),
-		Short: "shows a state_info by index or by height",
+		Use:   "state [rollapp-id]",
+		Short: "Query the state associated with the specified rollapp-id and any other flags. If no flags are provided, return the latest state.",
```

Comment:
add description for the `finalized` flag

Reply:
There is a description:
`cmd.Flags().Bool(FlagFinalized, false, "Indicates whether to return the latest finalized state")
`